app/controllers/spaces_controller.rb

```diff
@@ -36,10 +50,13 @@ def create
 
     if !space_params.has_key? :is_private
       @space.is_private = @space.user.prefers_private?
+      if @space.organization
+        @space.is_private = @space.is_private || organization.prefers_private?
```

Comment:
are we allowing organizations to have public models?  If not, I imagine we should stay simple here.